src/scss/core/_vars.scss

```diff
@@ -181,3 +181,11 @@ $tooltip-opacity: 1;
 // Cart Drawer
 // To be above zendesk help
 $zindex-cartdrawer: 9999991;
+
+// stockist
```

Comment:
this is a basic component, so please add the SELECT on FORM doc.
comment here should SELECT not STOCKIST.
Border-radius should be same value of form components 0.25rem